wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -225,6 +227,8 @@ namespace :wagn do
 
       Card.empty_trash
 
+      # FIXME: move this to history
```

Comment:
This should use a similar way of registering things to do with particular tasks, rather than having modules needing to customize rake tasks.